app/src/main/cpp/skyline/services/am/controller/ILibraryAppletCreator.cpp

```diff
@@ -1,9 +1,26 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
 
+#include <services/am/storage/IStorage.h>
+#include <services/am/applet/ILibraryAppletAccessor.h>
 #include "ILibraryAppletCreator.h"
 
 namespace skyline::service::am {
     ILibraryAppletCreator::ILibraryAppletCreator(const DeviceState &state, ServiceManager &manager) : BaseService(state, manager, Service::am_ILibraryAppletCreator, "am:ILibraryAppletCreator", {
+        {0x0, SFUNC(ILibraryAppletCreator::CreateLibraryApplet)},
+        {0xa, SFUNC(ILibraryAppletCreator::CreateStorage)}
     }) {}
+
+    void ILibraryAppletCreator::CreateLibraryApplet(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        manager.RegisterService(SRVREG(ILibraryAppletAccessor), session, response);
+    }
+
+    void ILibraryAppletCreator::CreateStorage(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        auto size = request.Pop<i64>();
```

Comment:
You might as well just read in an unsigned value as it being negative is just as invalid as it overflowing.

Reply:
Well overflowing in this case would still create the storage it would just be too big.